conf/app.routes

```diff
@@ -7,7 +7,8 @@ GET        /start                               uk.gov.hmrc.driverinspectionnoti
 GET        /search                              uk.gov.hmrc.driverinspectionnotificationfrontend.controllers.SearchController.show(gmrId: Option[String] ?= None)
 POST       /search                              uk.gov.hmrc.driverinspectionnotificationfrontend.controllers.SearchController.submit()
 
-GET        /results/:gmrId                      uk.gov.hmrc.driverinspectionnotificationfrontend.controllers.SearchResultController.result(gmrId: String)
+GET        /results/:gmrId                      uk.gov.hmrc.driverinspectionnotificationfrontend.controllers.SearchResultController.result(gmrId: String, checkedStatusAgain: Boolean = false)
+GET        /results/:gmrId/checked              uk.gov.hmrc.driverinspectionnotificationfrontend.controllers.SearchResultController.result(gmrId: String, checkedStatusAgain: Boolean = true)
```

Comment:
Wondering if we should use a query param rather than a different path? It feels like they're the same page and should have the same fundamental resource address.

Reply:
yeah, I wasn't too sure what would be best in this scenario. I've updated the PR. thanks @tipperba